src/types/global.d.ts

```diff
@@ -1,17 +1,27 @@
+import { PlayerManager } from "chromecast-caf-receiver/cast.framework";
+
+export interface DeviceInfo {
+    deviceId: string | number,
+    deviceName: string,
+    versionNumber: string
+}
+
+export interface GlobalScope {
+    [key: string]: any
+}
+
 declare global {
     export const PRODUCTION: boolean;
     export const RECEIVERVERSION: string;
+    export let $scope: GlobalScope;
```

Comment:
```suggestion
    export const $scope: GlobalScope;
```

Why not const?

Reply:
Ah my bad, I thought it was already const. I must've mixed it up with something else :p